src/main/java/com/google/sps/data/NSWGovAQDataPoint.java

```diff
@@ -17,6 +17,14 @@ public NSWGovAQDataPoint() {
     this.aqi = -1;
   }
 
+  public int getId() {
```

Comment:
Add constructor with args e.g. NSWGovAQDataPoint(int siteId, double aqi){} and setter method e.g. setSiteId(int siteId){} to allow you initialize an instance and update fields when needed.

Reply:
The pair of NSWGov* data classes are only instantiated by GSON conversion from the JSON strings returned by the NSW Gov API.

The GSON method only uses the no-args contructor, then applies reflection to set the values.

After that, these are only used to populate an AQDataPoint.

I would suggest that keeping these as strictly data objects used to convert from JSON, without any methods (beyond perhaps a toString() if that's useful) is the best approach.

Reply:
Yea, my impression isn't complete, thought there will be other use cases besides JSON conversion, 

```
I would suggest that keeping these as strictly data objects used to convert from JSON, without any methods (beyond perhaps a toString() if that's useful) is the best approach.
```
This makes sense.